### ISSUE TITLE
Integrate auto-generation of sprite images during build

### ISSUE THREAD

**Issue by jhpedemonte** [maqetta/maqetta#496]
See 575a0fc2351d2f787fef58a24a8f281a8c940690 for initial creation of sprite image, using http://spritegen.website-performance.org/.

However, adding or changing the sprite requires manual editing of sprite or going through the generation process again.  Easiest would be to keep the CSS files pointing to individual images, but do auto-generation of sprite image(s) during build process.  We can use a tool such as https://github.com/yostudios/Spritemapper.

**Comment by peller** [comment]
we shouldn't be editing these that often

**Comment by peller** [comment]
I'm wondering why the tool leaves so much space between the images?

**Comment by jhpedemonte** [comment]
Yes, Adam does have a point: we won't be doing this often, so maybe just keep it manual right now, and document the process so others can update images.

As for the space between images, I just took the default spacing by the tool.  With compression, though, it shouldn't make a difference.
